lib/portage/dbapi/vartree.py

```diff
@@ -2317,9 +2310,11 @@ def unmerge(
 
         if trimworld is not None:
             warnings.warn(
-                "The trimworld parameter of the "
-                + "portage.dbapi.vartree.dblink.unmerge()"
-                + " method is now unused.",
+                (
+                    "The trimworld parameter of the "
+                    "portage.dbapi.vartree.dblink.unmerge()"
+                    " method is now unused."
+                ),
```

Comment:
What does adding parentheses here gain us?

Reply:
For concatenating lines at the compile-time level rather than runtime with the `+` operator:

https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation

Reply:
Oh, I finally see what you mean. The parenthesis don't need to be there for that expression. Good to know! I'm fixing that right now.